clarity/src/vm/tests/simple_apply_eval.rs

```diff
@@ -1276,6 +1276,41 @@ fn test_stx_ops_errors() {
     }
 }
 
+#[test]
+fn test_bitwise() {
```

Comment:
I had tried tests with mismatching types here (e.g. that the 2nd parameter for `>>` and `<<` must be a uint, but the tests passed anyway.  However the type_checker tests succeed...  Do tests in this file skip type checking?

Reply:
`vm_execute_v2` (really an alias to the test function `execute_v2`) does not run the type-checker.  It just builds the AST and evaluates it.  For that, you'd use `analysis::run_analysis()`.

Reply:
Ah okay, thanks for the explanation 👍  I'll leave type checking out of this particular test then since it doesn't seem to be the use-case in this file.